ssh.go

```diff
@@ -68,29 +70,68 @@ func (s *SSHServer) handleTunnelCommand(sess ssh.Session) {
 	}
 }
 
-func (s *SSHServer) handleSessionWithCommand(sess ssh.Session) {
-	cmd := sess.Command()[0]
-	s.logger.Debugw("received command", "command", cmd)
+func (s *SSHServer) fillCommandsMap(commandsMap map[string]int, commands []string) {
 
-	cmdKey, cmdValue := s.parseCommand(cmd)
-	switch cmdKey {
-	case "tunnel":
-		shouldTunnel, err := strconv.ParseBool(cmdValue)
-		if err != nil {
-			s.logger.Errorw("failed to parse tunnel command", "error", err)
-			return
-		}
-		if !shouldTunnel {
-			s.handleBasicSession(sess)
-			return
+	commandsMap["ttl"] = MaxTtl //default value
+
+	for i := 0; i < len(commands); i++ {
```

Comment:
this `for` loop is nice when we want to handle multiple commands from the user, but we don't need it for now. If we do want to have keep this to be future-proof, then I would suggest creating a type for the `Command`, and not hardcode strings all over the place

Reply:
for now I suggest to remove the `commandsMap` (which is implicitly going to remove a lot of code, and resolve some comments I left), and to just handle the `ttl` as another case in the `switch` statement.